### DISCUSSION TITLE
shopee.co.id - Banner Cek Aplikasi

### DISCUSSION THREAD

**Discussion by 3xploiton3** [realodix/AdBlockID#1095] [Personal Filter]
**Jenis**

**Sample URL**
https://shopee.co.id/product/467299472/11255597199?smtt=0.11597773-1639733239.9
![Screenshot_20211217-162748862 (1)](https://user-images.githubusercontent.com/19517680/146523024-55023ff9-2081-479a-a8f8-8a2162548912.jpg)

**Penjelasan Lebih Lanjut**

**Comment by realodix** [comment]
Ini seharusnya masuke filter khusus Annoyance.

Sama seperti di forum, walaupun valid di HP, tapi Saya tidak bisa mereplikasinya di PC, jadi gak bisa bantu kasih kode

![noname-crop](https://user-images.githubusercontent.com/1314456/146551457-e5548f11-deef-46c0-bca7-a59dc27cac6a.png)

**Comment by 3xploiton3** [comment]
Saya sudah masukkan kode seperti di didiskusi tp sudah tidak bekerja

**Comment by realodix** [comment]
`shopee.co.id##a[href*="=newusrmblbanner"]`
